### ISSUE TITLE
Prevent browser popup when `Ctrl-S` / `Cmd-S` triggered

### ISSUE THREAD

**Issue by festoney8** [x1unix/go-playground#447]
**Is your feature request related to a problem? Please describe.**

`Ctrl-S` / `Cmd-S` will trigger a browser popup to save webpage, it's annoying, but it's hard to get out of that habit...

**Describe the solution you'd like**
- Just prevent event when `Ctrl-S` / `Cmd-S`
- Or change that shortcut to other function such as copy share url to clipboard, just like [typescript playground](https://www.typescriptlang.org/play/)

**Comment by x1unix** [comment]
@festoney8 if you would like to have a sharing snippet on save functionality - please update the issue description.

Also, I would like to get more feedback from a community on this feature request.

**Comment by festoney8** [comment]
For me, simply preventing the popup is enough.

I tested `Ctrl-S` in several playgrounds while creating this issue and found that this operation, both in TypeScript playground and official Go playground, is related to sharing.
Perhaps this is a more common practice? So I included that option in the solution. It's up to your decision.

I'd like to contribute but unfortunately I'm not familiar with react.
